circuits/web/tools.py

```diff
@@ -449,3 +451,20 @@ def gzip(response, level=4, mime_types=("text/html", "text/plain",)):
     return httperror(
         response.request, response, 406, description="identity, gzip"
     )
+
+
+class BehindProxy(BaseComponent):
+    http_index = ['X-Real-IP', 'X-Forwarded-For']
```

Comment:
Extra blank line between class and ``http_index``

Is ``headers`` a more clearer name? The list of headers we check?